README.md

```diff
@@ -212,7 +212,51 @@ Branch branch = Branch.getInstance(getApplicationContext(), "your test branch ke
 Either way, we recommend you put a `//TODO` to remind you to change back to live app during deployment later. 
 Also, note the Branch object is singleton, so you can and should still use `Branch.getInstance(getApplicationContext())` in all the other places (see examples below).
 
+#### Automatic Session Management
+Starting from 1.5.7, there is no need for intialising and closing session. Branch SDK can do it for you if you do one of the following options.
+
+1) If you are not creating an Application class, declare ```BranchApp``` as your application class in your manifest.
```

Comment:
For inline code, you can use single back-tick instead of three, e.g. `BranchApp`